packages/markuplint/src/reporter/github-reporter.ts

```diff
@@ -0,0 +1,28 @@
+import type { MLResultInfo } from '../types';
+import type { Severity } from '@markuplint/ml-config';
+
+import { messageToString } from '../util';
+
+export function githubReporter(results: MLResultInfo) {
+	const out: string[] = [];
+
+	for (const violation of results.violations) {
+		const command = severityToCommand(violation.severity);
+		const meg = messageToString(violation.message, violation.reason);
+		out.push(
+			`::${command} file=${results.filePath},line=${violation.line},col=${violation.col}::${meg} (${violation.ruleId})`,
+		);
+	}
+
+	return out;
+}
+
+function severityToCommand(severity: Severity) {
+	switch (severity) {
+		case 'info':
+			return 'notice';
```

Comment:
Convert to `notice` because GitHub does not support `info` messages.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-notice-message